docs/source/index.rst

```diff
@@ -14,7 +14,7 @@ Installation
 you must (Mac OS X default).
```

Comment:
I'd change this section to sth like:

```
``pip install blackbelt`` if you can. On macOS,

- ``brew install python`` and ``pip install blackbelt`` if you're old school,
- ``brew install python3`` and ``pip3 install blackbelt`` otherwise,
- ``easy_install blackbelt`` if there's no other option.
```

Reply:
👍

Reply:
Perhaps let's hold off for this until we can deem Python 3 support is no longer experimental, I'd give it a release in experimental status. This gives a chance for some people who use Python 3 to use Blackbelt in Python 3 and report any broken functionality, as it seems the test coverage isn't perfect.